api/dbConfig/init.js

```diff
@@ -0,0 +1,5 @@
+const { Pool } = require("pg");
+
+const pool = new Pool({ database: process.env.PGDATABASE});
```

Comment:
database: process.env.... is redundant you defined your env in your docker compose and don't have dependency to handle it